debezium-core/src/main/java/io/debezium/config/Configuration.java

```diff
@@ -2097,4 +2097,33 @@ default void forEachMatchingFieldName(Pattern regex, BiConsumer<String, String>
     default <T> void forEach(BiConsumer<String, String> function) {
         this.asMap().forEach(function);
     }
+
+    /**
+     * Returns the string config value from newProperty config field if it's set or its default value when it's not
+     * set/null.If both are null it returns the value of the oldProperty config field, or its default value when it's
+     * null.
+     * This fallback only works for newProperty fields that have a null / not-set default value!
+     *
+     * @param newProperty the new property config field
+     * @param oldProperty the old / fallback property config field
+     * @return the evaluated value
+     */
+    default String getFallbackStringProperty(Field newProperty, Field oldProperty) {
+        return getString(newProperty, () -> getString(oldProperty));
```

Comment:
I didn't check, do we log a warning somewhere when using the old names?

Reply:
no, I will add logging to the lambda.

Reply:
Done